fec.config.js

```diff
@@ -26,15 +27,23 @@ module.exports = {
     }),
   },
   moduleFederation: {
-    moduleName: 'widgetLayout',
+    moduleName: process.env.CONFIG_PORT || process.env.PROXY ? 'frontendStarterApp' : 'widgetLayout', // overwriting starter app locally for ease of development
     exposes: {
       './RootApp': path.resolve(__dirname, './src/AppEntry.tsx'),
+      // TODO these will likely need to be restructured to be more consumable
+      './WidgetLayout': path.resolve(__dirname, './src/Components/DnDLayout/GridLayout.tsx'),
+      './WidgetDrawer': path.resolve(__dirname, './src/Components/WidgetDrawer/WidgetDrawer.tsx'),
+      './WidgetHeader': path.resolve(__dirname, './src/Components/Header/Header.tsx'),
     },
-    exclude: ['react-router-dom', 'jotai'],
+    exclude: ['@patternfly/react-core', 'jotai', 'react', 'react-dom', 'react-redux', 'react-router-dom'],
     shared: [
       {
-        'react-router-dom': { singleton: true, requiredVersion: '*' },
+        '@patternfly/react-core': { singleton: true, requiredVersion: dependencies['@patternfly/react-core'] },
         jotai: { singleton: true, requiredVersion: '*' },
+        react: { singleton: true, requiredVersion: dependencies.react },
+        'react-dom': { singleton: true, requiredVersion: dependencies['react-dom'] },
+        'react-redux': { singleton: true, requiredVersion: dependencies['react-redux'] },
+        'react-router-dom': { singleton: true, requiredVersion: dependencies['react-router-dom'] },
```

Comment:
Do not list packages that are not used. (Redux)

Reply:
@Hyperkid123 `react-redux` is used as AFAIK as the FEC notifications package seems to depend on it? See:
https://github.com/RedHatInsights/widget-layout/blob/master/src/Components/DnDLayout/GridLayout.tsx#L26
https://github.com/RedHatInsights/widget-layout/blob/master/src/store/index.ts#L11
https://github.com/RedHatInsights/widget-layout/blob/master/src/AppEntry.tsx

Or are you saying because redux is configured on the page for us by chrome - we should only have this for development?